bundles/config/org.eclipse.smarthome.config.core/src/main/java/org/eclipse/smarthome/config/core/validation/internal/ConfigDescriptionParameterValidator.java

```diff
@@ -12,20 +12,30 @@
 
 /**
  * The {@link ConfigDescriptionParameterValidator} can be implemented to provide a specific validation of a
- * {@link ConfigDescriptionParameter} and its actual value.
+ * {@link ConfigDescriptionParameter} and its value to be set.
  *
  * @author Thomas Höfer - Initial contribution
  */
 public interface ConfigDescriptionParameterValidator {
 
     /**
+     * Returns true, if the validator is to be executed for the given parameter and its given value, otherwise false.
+     *
+     * @param parameter the corresponding config description parameter
+     * @param value the value to be set for the config description parameter
+     *
+     * @return true, if the validator is to be executed for the given parameter and its given value, otherwise false
+     */
+    boolean isExecutable(ConfigDescriptionParameter parameter, Object value);
```

Comment:
> "I doesn't validate anything, because I don't care"

Such a function should then imho not have the value passed in at all, because as soon as you evaluate the value, you automatically care. That's why I think the current implementation is confusing - you already do a validation within the isExecutable method and in validate, only the validation message is determined.